### ISSUE TITLE
sitewide 508-defect-2 [COGNITION, FOCUS MANAGEMENT]: Feedback button should have a focus halo when focused

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#17460]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Jennifer

## User Story or Problem Statement

As a keyboard user, I expect to tab through the focusable objects on a screen, so that I may activate content on the screen.

## Details

On pages where the new Feedback button appears, when a user tabs to the button the focus halo is not displayed. The focus styling should be used here, consistent across VA.gov.

## Acceptance Criteria

- [ ] When the Feedback button has focus, the VA.gov focus styling is visible

## Environment

* Operating System: all
* Browser: all
* Server destination: staging

## Steps to Recreate

1. Enter `https://staging.va.gov/find-locations` in browser
2. Verify that the Feedback button is visible on the screen
3. Tab through the screen
4. Verify that it isn't possible to see a focus halo on the Feedback button

## WCAG or Vendor Guidance (optional)

* [WCAG 2.0 Level AA - 2.4.7](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html) - Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible. 

## Screenshots or Trace Logs

**This image shows that the Feedback button is the active element in focus, and that the focus halo is not visible.**

![Screen Shot 2020-12-14 at 4 07 49 PM](https://user-images.githubusercontent.com/57469/102137720-45c43580-3e29-11eb-999d-4e1cb1e3973b.png)

**Comment by joanneesteban** [comment]
Thanks for flagging, @jenniferlee-dsva . This issue is captured in the [Medallia accessibility review](https://github.com/department-of-veterans-affairs/va.gov-team/issues/15214). FYSA, @jwoodman5 .

**Comment by jenstrickland** [comment]
Ah, sorry! I didn't see an issue ticket for it.

**Comment by joanneesteban** [comment]
Np, @jenstrickland ! I very much appreciate the Accessibility push!

**Comment by billfienberg** [comment]
It looks like the user can tab to the elements behind the modal when the modal is open. 

![medallia-tab-bug](https://user-images.githubusercontent.com/6130520/104759168-215bc180-5725-11eb-830f-4e15758b81c2.gif)

**Comment by jwoodman5** [comment]
We will make sure this is addressed prior to launching the Feedback form. Thanks.

**Comment by Kimberley2019** [comment]
@jwoodman5

**Comment by jwoodman5** [comment]
@billfienberg Going through tix in the backlog today and saw this again. It doesn't seem to be specifically called out in our 508 issue sheet for Medallia. When testing this, a focus halo appears but it is not consistent with the rest of the page (gray vs. gold/yellow). Do you have any other context on this? Thanks.

**Comment by billfienberg** [comment]
@jwoodman5, good call. This issue is not included in https://github.com/department-of-veterans-affairs/va.gov-team/issues/18706 or the tracker spreadsheet that Sarkis created. I'll add this to my todo list for today.

**Comment by billfienberg** [comment]
@jwoodman5, I just opened a [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/16158) to update the focus styles of the Feedback button on https://staging.va.gov/find-locations.

**Comment by billfienberg** [comment]
My [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/16158) has been deployed to staging, it looks like the feedback button on https://staging.va.gov/find-locations has the correct focus styles. 

![medallia-feedback-button-has-updated-focus-style-on-staging](https://user-images.githubusercontent.com/6130520/109074903-03876200-76be-11eb-965e-adacc8e4745d.gif)

**Comment by billfienberg** [comment]
@jwoodman5, I think this ticket is ready for sign-off and/or closing.

**Comment by jwoodman5** [comment]
Fantastic!! Thanks for the quick attention to this @billfienberg

**Comment by jwoodman5** [comment]
Tested myself on staging as well and looks good. Closing.
